curriculum/challenges/english/07-scientific-computing-with-python/learn-list-comprehension-by-building-a-case-converter-program/657ef2a86d4e545cec9a85fb.md

```diff
@@ -9,7 +9,9 @@ dashedName: step-4
 
 Uppercase characters in camel case or pascal case indicate the start of new words.
 
-Inside the loop body, use an `if` statement in conjunction with the `.isupper()` string method to check for uppercase characters and move `pass` inside the new `if` statement.
+Inside the loop body, replace the `pass` statement with an `if` statement in conjunction with the `.isupper()` string method to check for uppercase characters.
+
+For now, add a `pass` statement in the if statement body.
```

Comment:
I wouldn't change the original sentence. It should be pretty clear that `pass` needs to be moved inside the new `if`.